spec-tests/src/test/java/org/candlepin/spec/CloudRegistrationSpecTest.java

```diff
@@ -447,6 +453,155 @@ public void shouldNotExportCertificatesWithAnonymousTokenForUnknownAnonCloudCons
+        assertThatStatus(() -> ApiClients.bearerToken(result.getToken()).consumers()
+            .createConsumer(Consumers.random(ownerDTO)))
+            .isTooMany()
+            .hasHeaderWithValue("retry-after", 30);
+
+        // Product and owner exist in Candlepin
+        adminClient.ownerProducts().createProductByOwner(ownerDTO.getKey(), productDTO);
+        adminClient.owners().createPool(ownerDTO.getKey(), Pools.random(productDTO));
+        ConsumerDTO consumer = ApiClients.bearerToken(result.getToken()).consumers()
+            .createConsumer(Consumers.random(ownerDTO));
+
+        assertThat(consumer)
+            .isNotNull()
+            .extracting(ConsumerDTO::getIdCert)
+            .isNotNull();
+    }
+
+    /**
+     * Associates a cloud offering ID to product IDs in the hosted test adapters.
+     *
+     * @param client
+     *     client used to make the request to the hosted test endpoint
+     *
+     * @param cloudOfferId
+     *     the offering ID to associate to a product ID
+     *
+     * @param productIds
+     *     the product IDs to associate to an offering ID
+     */
+    private void associateProductIdToCloudOffer(ApiClient client, String cloudOfferId,
+        Collection<String> productIds) {
+        ObjectNode objectNode = ApiClient.MAPPER.createObjectNode();
+        objectNode.put("cloudOfferId", cloudOfferId);
+
+        ArrayNode productsNode = ApiClient.MAPPER.createArrayNode();
+        productIds.forEach(prod -> productsNode.add(prod));
+        objectNode.putPOJO("productIds", productsNode);
+
+        Response response = Request.from(client)
+            .setPath("/hostedtest/cloud/offers")
+            .setMethod("POST")
+            .setBody(objectNode.toString())
+            .execute();
+
+        if (response.getCode() != 204) {
+            throw new RuntimeException("unable to associate product to cloud offer");
+        }
+    }
+
+    /**
+     * Associates a cloud account ID to an owner ID in the hosted test adapters.
+     *
+     * @param client
+     *     client used to make the request to the hosted test endpoint
+     *
+     * @param cloudAccountId
+     *     the cloud account ID to associate to an owner ID
+     *
+     * @param ownerId
+     *     the owner ID to associate to a cloud account ID
+     */
+    private void associateOwnerToCloudAccount(ApiClient client, String cloudAccountId, String ownerId) {
+        ObjectNode objectNode = ApiClient.MAPPER.createObjectNode();
+        objectNode.put("cloudAccountId", cloudAccountId);
+        objectNode.put("ownerId", ownerId);
+
+        Response response = Request.from(client)
+            .setPath("/hostedtest/cloud/accounts")
+            .setMethod("POST")
+            .setBody(objectNode.toString())
+            .execute();
+
+        if (response.getCode() != 204) {
+            throw new RuntimeException("unable to associate cloud account to owner");
+        }
+    }
```

Comment:
This was moved into the HostedTestAPI so you can just use adminClient.hosted().associateOwnerToCloudAccount instead

Reply:
done

Reply:
You should be able to remove this method now that it is not being used!

Reply:
done